### DISCUSSION TITLE
Abour Reclamation Algorithm.

### DISCUSSION THREAD

**Discussion by thepeppapower** [MaaAssistantArknights/MaaAssistantArknights#10256] [Q&A] [answered]
Wanted to ask about this mode for EN servers. Are there any news when it is going to be featured? Plus, I wanted to ask about whether there is some test versions to try? Thanks in advance)

**Comment by Constrat** [comment, marked as answer]
Nightly is always available to try. It's in the release/tags section.
Neither stable nor beta still have it as they were last updated 2 weeks ago.
